src/menu/src/Menu.js

```diff
@@ -34,11 +34,9 @@ export default class Menu extends React.PureComponent {
 
   componentDidMount() {
     // Get the menu item buttons
-    this.menuItems = [
-      ...this.menuRef.querySelectorAll(
-        '[role="menuitemradio"], [role="menuitem"]'
-      )
-    ]
+    this.menuItems = Array.prototype.slice.call(
```

Comment:
I think we could use `Array.from` instead

Reply:
I'm guessing the original author wrote `Array.from` as well but one of the pre-commit hook (xo) just auto changes/"fixes" it to use spread operator instead, and we're back to the original transpilaton issue.

Reply:
Would making it a two step process prevent the transpilaton issue: 
```javascript
    // Get the menu item buttons
    const menuItemsNodes = this.menuRef.querySelectorAll('[role="menuitemradio"], [role="menuitem"]');

    // Convert to an array
    this.menuItems = [ ...menuItemsNodes ];
```

Reply:
@bchelli The issue is with the spread itself. Gatsby doesn't realize `menuItemsNodes` is array-like. This is what the result looks like

```javascript
var menuItemsNodes = this.menuRef.querySelectorAll('[role="menuitemradio"], [role="menuitem"]');
this.menuItems = menuItemsNodes.concat();
```

which is invalid. Perhaps an issue with the gatsby/babel configs for docs.

Reply:
@sumanbh is right that some hook changed the code. I initially wrote `Array.from` this also happened in a different case. Not sure what the right solution here is. @Rowno any ideas?

Reply:
![image](https://user-images.githubusercontent.com/564463/47190365-c775b280-d2f5-11e8-9e30-7ed096ac8516.png)

Reply:
@sumanbh might be nice to have update this too  in `./src/table/src/manageTableCellFocusInteraction.js` and `./src/table/src/manageTableRowFocusInteraction.js`

Reply:
@jeroenransijn Updated.

Reply:
I think we can just change it back to `Array.from()` and add an `// eslint-disable-next-line unicorn/prefer-spread`

Reply:
@Rowno Done.